core/merginapi.cpp

```diff
@@ -3222,8 +3228,8 @@ ProjectDiff MerginApi::compareProjectFiles(
   /*
   for ( MerginFile file : oldServerFilesMap )
   {
-    // R-D/L-D
-    // TODO: need to do anything?
+  // R-D/L-D
+  // TODO: need to do anything?
```

Comment:
I would rather keep the old formatting so it does not show as your change in the history :)